fetpype/workflows/pipeline_fet.py

```diff
@@ -231,9 +231,16 @@ def create_main_workflow(
 
     main_workflow.connect(
         fet_pipe,
-        "outputnode.output_surf",
+        "outputnode.output_surf_lh",
         surf_datasink,
-        f"@{cfg.surface.pipeline}",
+        "@surf_lh",
+    )
+
+    main_workflow.connect(
```

Comment:
I guess you know, but depending on how the output file is named for the surface extraction, we will need to adapt the datasink to parse and rename the file as desired for this output. This is missing for now, but I can add it if you want, let me know :)

Reply:
@t-sanchez I have modifed the data sink and it is working for both surf.gii and .stl, here is a capture of the results
<img width="1836" height="48" alt="Capture d’écran du 2025-09-03 13-56-37" src="https://github.com/user-attachments/assets/68936732-a950-43f0-89c5-1e3950c165d6" />
If you are happy with that you can accept the PR (pls keep only one line in the squash and merge)